lib/Service/FontService.php

```diff
@@ -242,4 +243,41 @@ private function generateFontOverview(ISimpleFile $fontFile): void {
 			// in the UI and display a fallback message
 		}
 	}
+
+	/**
+	 * @throws Exception
+	 */
+	public function installDefaultFonts(): void {
+		$dirPath = __DIR__ . '/../../assets/fonts';
+
+		if (!is_dir($dirPath)) {
+			throw new Exception("Directory \"$dirPath\" does not exist!");
+		}
+
+		$handle = opendir($dirPath);
+
+		if (!$handle) {
+			throw new Exception("Failed opening directory \"$dirPath\"!");
+		}
+
+		while (false !== ($fileName = readdir($handle))) {
+			if ($fileName === '.' || $fileName === '..') {
+				continue;
+			}
+
+			$filePath = $dirPath . '/' . $fileName;
+
+			if (!is_file($filePath)) {
+				continue;
+			}
+
+			$fileContent = file_get_contents($filePath);
```

Comment:
While this is fine for this case a general note that using file_get_contents would read the full file content into memory which might cause problems with larger files. Better approach would be to use fopen instead and pass a file pointer instead which is also supported by the `newFile` method that is called within `uploadFontFile`